packages/forms/src/components/RichTextInput/RichTextRenderer.tsx

```diff
@@ -55,12 +56,21 @@ const ListItemNode: NodeRenderer = ({ children }) => {
   return <li>{children}</li>;
 };
 
+const HeadingNode: NodeRenderer = ({ children }) => {
+  return (
+    <Heading level="h3" size="h4">
```

Comment:
Why is the level and the size different?

Reply:
Sadly, because we have yet to get to [this ticket to fix our heading size](https://github.com/GCTC-NTGC/gc-digital-talent/issues/11859)s :confounded: